src/LinAlg/SAM.h

```diff
@@ -54,6 +54,10 @@ class SAM
   int getNoDOFs() const { return ndof; }
   //! \brief Returns the number of equations (free DOFs) in the model.
   virtual int getNoEquations() const { return neq; }
+  //! \brief Returns the number of equations for a given dofType.
+  //! \param nodeType Type of node
```

Comment:
The korrekte er vel "Returns the equation numbers for a ...". Number of equations leser jeg som antall ligninger (ett enkelt tall) ikke ligningsnumrene som sådan.